src/SearchModal.tsx

```diff
@@ -37,20 +41,132 @@ export const SearchModal = forwardRef(({ onClose, isOpen, children }: Props, ref
               onEscapeKey={onCloseDelayed}
               onClickOutside={onCloseDelayed}
               // Enable manual focus return to work.
-              returnFocus={false}
+              returnFocus={true}
```

Comment:
Is this supposed to return focus to the button when the modal is closed? I noticed that this isn't working in this PR, but it is working in master. So I think there's a regression regarding focus in this PR, potentially related to this change.

Reply:
Oops I was trying something out here and forgot to take it out. Good catch!

Reply:
[a71ff14](https://github.com/vectara/react-search/pull/47/commits/a71ff1453c0f101c7d24a21d409f0f210ae0ac6d)